ci/build-msys2.sh

```diff
@@ -8,9 +8,20 @@ args=(
   -D{egl-angle-lib,egl-angle-win32,pdf-build,rubberband,win32-smtc}=enabled
 )
 
-[[ "$SYS" == "clang64" ]] && args+=(
-  -Db_sanitize=address,undefined
-)
+if [[ "$SYS" == "clang64" ]]; then
+    args+=(
+      -Db_sanitize=address,undefined
+    )
+else # sanitizers are not supported on stable rust yet so clang64 is excluded
+    echo "::group::Building subrandr"
+    build_subrandr "$PWD/subrandr_prefix"
+    echo "::endgroup::"
+    export PKG_CONFIG_PATH="$PWD/subrandr_prefix/lib/pkgconfig:$PKG_CONFIG_PATH"
+    mkdir build
+    # make sure mpv.com will be able to find the DLL
+    cp subrandr_prefix/bin/subrandr-[0-9]*.dll build
```

Comment:
Could we use `LD_LIBRARY_PATH` instead? Or actually, just do a full install of subrandr?

Reply:
I think I tried using `LD_LIBRARY_PATH` initially but then realized this is Windows and that's not how the linker works here.
A full install would also work, as long as installing stuff to like `/usr/local` works on msys2.